### ISSUE TITLE
Add option to use `--tls-no-verify`

### ISSUE THREAD

**Issue by pavelzw** [prefix-dev/pixi#327]
### Problem description

It would be nice to have the option to ignore the SSL certificates if they are not trusted.

**Comment by ruben-arts** [comment]
Yep, its on the board! I think this would also fit with a big improvement on all of the authentication issues.

**Comment by jpursell** [comment]
I would also like this feature

**Comment by wolfv** [comment]
I am making a separate PR for global configuration where we can add the config options for this.
